githubscript/src/githubscript/script.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+""" Github main script
+"""
+import logging
+import os
+
+from githubscript.github import release
+from githubscript.release_config import get_release_config
+from scriptworker_client.client import sync_main
+from scriptworker_client.exceptions import ClientError, TaskVerificationError
+from scriptworker_client.utils import get_single_item_from_sequence
+
+log = logging.getLogger(__name__)
+
+
+async def async_main(config, task):
+    project_config = _get_project_config(config, task)
+    release_config = get_release_config(project_config, task["payload"], config)
+    contact_server = bool(config.get("contact_server"))
+
+    _log_warning_forewords(contact_server)
+
+    action = task["payload"]["action"]
+    if action == "release":
```

Comment:
Once we have multiple actions and repos supported, we may want to limit which actions are available on which repos, and require specific scopes for some of them.

Reply:
Done. It's now part of scopes and products have to allow this specific action in the config.